File: 2. Loop Problems/Loop problems in python/p21.py


n = int(input("Enter n: "))

a= 1
b= 2
print("Fibonacci Sequence: ", end= " ")

for i in range(1,n+1):
    print(f"{a}",end=" ")
    next = a + b
    a = b
    b = next
    
    

File: 2. Loop Problems/Loop Problems in c/p21.c
//  Write a C program to print Fibonacci series up to n terms.


#include<stdio.h>

int main() {
    int n,num1,num2,temp;
    printf("Enter n: ");
    scanf("%d",&n);


    num1= 1;
    num2= 1;

    printf("Fibonacci Sequence: ");

    for (int i=1; i <=n;i++) {

        printf("%d ",num1);
        
        temp = num1 + num2;
        num1 = num2;
        num2 = temp;
    }
}